src/test/java/com/reservation/studio/domain/BusinessHoursTest.java

```diff
@@ -0,0 +1,91 @@
+package com.reservation.studio.domain;
+
+import com.reservation.studio.domain.exception.InvalidBusinessHoursSizeException;
+import com.reservation.studio.domain.exception.MissingBusinessHoursScheduleException;
+import org.junit.jupiter.api.Test;
+
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+import java.util.EnumMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class BusinessHoursTest {
+
+    @Test
+    void shouldCreateBusinessHoursWithAllDaysAndValidSchedules() {
+        BusinessHours businessHours = new BusinessHoursTestBuilder()
+                .openAllDays(LocalTime.of(9, 0), LocalTime.of(17, 0))
+                .build();
+
+        for (DayOfWeek day : DayOfWeek.values()) {
+            assertThat(businessHours.scheduleFor(day)).isInstanceOf(OpeningSchedule.class);
```

Comment:
dodałbym jeszcze isNotNull()

```suggestion
            assertThat(businessHours.scheduleFor(day))
                    .isNotNull()
                    .isInstanceOf(OpeningSchedule.class);
```